### PR TITLE
Update application-deployment-via-mecm.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the Prerequisites as higher instead of older

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10706

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/application-deployment-via-mecm.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ This article guides you in migrating down-level servers from Microsoft Monitorin
 
 ## Prerequisites
 
-- Microsoft Endpoint Configuration Manager (MECM) older than 2207.
+- Microsoft Endpoint Configuration Manager (MECM) higher than 2207.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/application-deployment-via-mecm.md:34]:
```suggestion
- Microsoft Endpoint Configuration Manager (MECM) lower than version 2207.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review, @msbemba, I am not sure about his, is it lower or higher than version 2207?

File: microsoft-365/security/defender-endpoint/application-deployment-via-mecm.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/security/defender-endpoint/application-deployment-via-mecm.md:34]: It is higher than 2207.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@alekyaj Please merge this PR. Thanks

**Comment by alekyaj** [comment]
> @alekyaj Please merge this PR. Thanks

Hi @yogkumgit, I don't have the necessary access to merge this PR. But, I will assign the sign-off label so that PubOps can review and merge it. Thanks for reminding.

**Comment by yogkumgit** [comment]
@alekyaj Thanks. Wil wait for signoff label.

**Comment by alekyaj** [comment]
> Made changes to the Prerequisites as higher instead of older

@yogkumgit , this seems to be Public repo and hence I cannot find the sign-off label for the PubOps team to review and merge. I am duplicating the same change in https://github.com/MicrosoftDocs/microsoft-365-docs-pr/pull/19059. You can close this PR, if you wish. I've applied sign-off label on the duplicated PR I mentioned here, which will will be merged tonight and the changes will be live soon after that. Thanks!

**Comment by msbemba** [comment]
@yogkumgit  This can be closed as the duplicate PR is merged.

**Pull request closed without merging**